restapi/configure_cludod.go

```diff
@@ -92,6 +92,25 @@ func configureAPI(api *operations.CludodAPI) http.Handler {
 					return &principalID, nil
 				}
 			}
+		} else {
+			authz, err := conf.Server.NewGithubAuthorizer()
+			if err != nil {
+				api.Logger("ERROR: Failed to create authorizer: %v", err)
+				return nil, errors.New(500, "Failed to create authorizer: %v", err)
+			}
+			id, ok, err := authz.CheckAuthHeader(token)
+			if err != nil {
+				api.Logger("ERROR: Failed to validate message signature: %v", err)
```

Comment:
Also improving logging: let's make it clear where the message signature is coming from.  Is this from Github?  What's the result?  Are we failing to authorize a request?  We should log this as well.

Reply:
I improved these messages a bit, but honestly I am only now starting to dig into the details of what is going on with the SSH code that @robotoer wrote and the logs in general probably need some attention and creation of log levels, etc, so that we can do a better job of raising up user facing errors, but also keeping the detail for devs.

Reply:
ah, didn't realize these were moved around - thought you added this log message. sorry!